wagtailio/static/sass/layout/_header.scss

```diff
@@ -8,43 +8,75 @@
     position: relative;
 
     @include media-query(medium) {
+        padding: 35px 50px;
         margin: 0 auto;
-        padding: 15px 30px;
     }
 
-    @include media-query(large) {
-        padding: 35px 50px;
+    &__actions {
+        display: flex;
+        align-items: center;
+        position: relative;
+        top: -7px;
+
+        @include media-query(medium) {
+            top: -7px;
```

Comment:
Question: can this be removed as it's the same as the line 19?

Reply:
oops, sure can, thanks